src/v2/cookbook/unit-testing-vue-components.md

```diff
@@ -83,11 +84,12 @@ Automated testing allows large teams of developers to maintain complex codebases
 
 #### Getting started
 
-[vue-test-utils](https://github.com/vuejs/vue-test-utils) is the official library for unit testing Vue components. The [vue-cli](https://github.com/vuejs/vue-cli) webpack template comes with either Karma or Jest, both well supported test runners, and there are some [guides](https://vue-test-utils.vuejs.org/en/guides/) in the `vue-test-utils` documentation.
+[Vue Test Utils](https://github.com/vuejs/vue-test-utils) is the official library for unit testing Vue components. The [Vue CLI](https://github.com/vuejs/vue-cli) `webpack` template comes with either Karma or Jest, both well supported test runners, and there are some [guides](https://vue-test-utils.vuejs.org/en/guides/) in the Vue Test Utils documentation.
```

Comment:
vue-cli refers to itself as vue-cli on the readme, not Vue CLI. This is pedantic, I know, but I think the rule is that we refer to the project based on how it refers to itself.

Reply:
Oh, yes you are right.
Actually I had a small discussion about Title Cases for our all package names in docs (except their npm package names), and the CLI is missed by me. 😅
Refs: https://github.com/vuejs/vue-test-utils/pull/443 https://github.com/vuejs/vue-loader/pull/1176
But no worries, I will revert it first until we have a result about the name. 🙂